app-ios/README.md

```diff
@@ -10,6 +10,14 @@ cd ../
  open DroidKaigi2022/DroidKaigi2022.xcodeproj
 ```
 
+## Try it out
```

Comment:
I think you can add it in README.

## Try it out
### Android
...
### iOS
....